maestro-android/src/androidTest/java/dev/mobile/maestro/MaestroDriverService.kt

```diff
@@ -170,6 +173,22 @@ class Service(
         responseObserver.onCompleted()
     }
 
+    override fun screenshot(
+        request: MaestroAndroid.ScreenshotRequest,
+        responseObserver: StreamObserver<MaestroAndroid.ScreenshotResponse>
+    ) {
+        val outputStream = ByteString.newOutput()
+        val bitmap = uiAutomation.takeScreenshot()
```

Comment:
What advantage this has over adb?

Reply:
This handles the screenshot all in memory and doesn't require writing to a file on the device. Also removes a dependency on the screencap tool